jsonrpc/http.go

```diff
@@ -19,14 +21,25 @@ type HTTP struct {
 	rpc      *Server
 	log      utils.SimpleLogger
 	listener net.Listener
+
+	// metrics
+	requests prometheus.Counter
 }
 
 func NewHTTP(listener net.Listener, rpc *Server, log utils.SimpleLogger) *HTTP {
-	return &HTTP{
+	h := &HTTP{
 		rpc:      rpc,
 		log:      log,
 		listener: listener,
+
+		requests: prometheus.NewCounter(prometheus.CounterOpts{
+			Namespace: "rpc",
+			Subsystem: "http",
+			Name:      "requests",
```

Comment:
let's make it more specific `http request` and the same for WebSockets.

Reply:
It has `Subsystem: "http"` already. WS has `Subsystem: "ws",`